packages/flutter/lib/src/material/slider.dart

```diff
@@ -837,8 +837,11 @@ class _SliderRenderObjectWidget extends LeafRenderObjectWidget {
   @override
   void updateRenderObject(BuildContext context, _RenderSlider renderObject) {
     renderObject
-      ..value = value
+      // We should update the `divisions` ahead of `value`, because the `value`
```

Comment:
Since the regression tests link to this issue, this comment and the one in `range_slider.dart`arent' strictly necessary

Reply:
Removed.